sql/builtin_functions.cc

```diff
@@ -1777,6 +1778,30 @@ BoundFunction norm(const std::vector<BoundSqlExpression> & args)
 static RegisterBuiltin registerNorm(norm, "norm");
 
 
+struct ParseJsonOptions {
+    bool ignoreErrors = false;
+    JsonArrayHandling arrays = PARSE_ARRAYS;
+};
+
+DECLARE_STRUCTURE_DESCRIPTION(ParseJsonOptions);
+DEFINE_STRUCTURE_DESCRIPTION(ParseJsonOptions);
+
+ParseJsonOptionsDescription::
+ParseJsonOptionsDescription()
+{
+    addAuto("ignoreErrors", &ParseJsonOptions::ignoreErrors,
+            "If true, errors in the JSON are ignored and the element with "
+            "an error will be silently ignored.  If false (the default), "
+            "a JSON format error will lead to the function failing with "
+            "an exception.");
```

Comment:
In that case, you should adjust the DOD for the try function to include replacing this functionality, and include it there.  This is just a refactoring that keeps the same options as before; modifying the interface requires coordination with customers and should be handled in a separate ticket.